src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -66,8 +67,8 @@
              let-actionDescription="actionDescription">
   <div *ngIf="!safeToDestroyResult['is_safe_to_destroy']"
        class="danger">
-    <cd-warning-panel i18n>The OSD is not safe to destroy!</cd-warning-panel>
+    <cd-warning-panel i18n>The OSD(s) is/are not safe to destroy!</cd-warning-panel>
   </div>
-  <ng-container i18n><strong>OSD {{ selection.first().id }}</strong> will be
+  <ng-container i18n><strong>OSD {{ selection.getIds().toString() }}</strong> will be
```

Comment:
Use lodash to get the ID's of the selected OSD's and the the ``list`` pipe to format it nicely. Don't forget to make ``_`` public, otherwise you can not access lodash in the template file.

```suggestion
  <ng-container i18n><strong>OSD {{ _.map(selection.selected, (s) => s.id) | list }}</strong> will be
```

Reply:
Alternatively you could implement a new pipe that works like the [Jinja2 attr filter](https://jinja.palletsprojects.com/en/2.10.x/templates/#attr).

```<ng-container i18n><strong>OSD {{ selection.selected | attr:'id' | list }}</strong> will be`